shell/mixins/resource-fetch.js

```diff
@@ -31,11 +31,18 @@ export default {
       perfConfig = DEFAULT_PERF_SETTING;
     }
 
+    // Normally owner components supply `resource` and `inStore` as part of their data, however these are needed here before parent data runs
+    // So set up both here
+    const params = { ...this.$route.params };
+    const resource = params.resource || this.schema?.id; // Resource can either be on a page showing single list, or a page of a resource showing a list of another resource
```

Comment:
Is this for instances like being on a Node detail page and we're listing pods? If so, can we just make a pods list component and just use this on the components that are going to list paginated resources and not have to differentiate between list/detail page in a mixin like this?